src/spdx3/bump_from_spdx2/annotation.py

```diff
@@ -24,4 +26,4 @@ def bump_annotation(spdx2_annotation: Spdx2_Annotation, creation_info: CreationI
     subject: str = spdx2_annotation.spdx_id
     statement: str = spdx2_annotation.annotation_comment
 
-    return Annotation(spdx_id, creation_info, annotation_type, subject, statement=statement)
+    payload.add_element(Annotation(spdx_id, creation_info, annotation_type, subject, statement=statement))
```

Comment:
Just for the record: Inititally this implementation should have been "target-based", so the return value should be the element to create. With the upcoming Agent  we could still use this approach but with the problem of inlining the Agent Element and problems due to circular import we would need to return the Agent additionally. I think its okay to simply add the bumped element here and make all these bump methods dependant of the payload, so we will always need a `Payload` object to bump any other element from SPDX2.x. As I think that this seems to be the bigger picture for SPDX3.0, I am fine with thesse changes.